portfolio/src/main/webapp/index.html

```diff
@@ -9,34 +9,82 @@
-          Growing up in Miami, a city centered around cars, I am fascinated with the urban-planning model called superblocks that are being implemented in Spain! 
+          During my time at Northeastern I joined a salsa/bachata dance club called Kaliente in 
+          order to maintain some resemblace of home while I was away studying in Boston. 
+          Growing up in Miami, a city centered around cars, I am fascinated with the urban-planning 
+          model called superblocks that are being implemented in Spain! 
           I hope to one day travel to Spain and experience their culture rich cities.
       </p>
+      </div>
 
       <h3>Projects</h3>
-      <a target="_blank" href="https://github.com/kev1n80">Visit my Github!</a>
-      <a target="_blank" href="https://www.linkedin.com/in/kevin-cam">Visit my LinkedIn!</a>
-      <ul>
-          <li>Schedify</li>
-          <li>Pharmd-tracker</li>
-      </ul>
+        <div id="project-container"
+        <ul>
+            <li>Schedify</li>
+            <li>Pharmd-tracker</li>
+        </ul>
+      </div>
+
       <h3>Photography</h3>
-      <p>I love to offer my perspective of the world around me through photography. During quarantine I have found that you can offer magical experiences through mundane objects.</p>
-      <img src="https://scontent-mia3-2.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/104299695_3065854933496343_1464411969370242575_n.jpg?_nc_ht=scontent-mia3-2.cdninstagram.com&_nc_cat=103&_nc_ohc=Jca8JZrp1tkAX_kJhYl&oh=e99d5e202bb90fcc5eaed3baa60a5821&oe=5F1BCB3D"
-            alt="sunset in a cup of coffee">
-      <img src="https://scontent-mia3-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/101156385_128157258893640_5906348761081260841_n.jpg?_nc_ht=scontent-mia3-1.cdninstagram.com&_nc_cat=100&_nc_ohc=H0f09Iq1QT0AX9ijces&oh=0efa58121deb15f6034a217b9a1b2915&oe=5F1A5C33"
-            alt="Luke Skywalker using the force">
+      <p>I love to offer my perspective of the world around me through photography. During 
+          quarantine I have found that you can offer magical experiences through mundane 
+          objects.</p>   
+      <div id="album">     
+        <div class="photo-container">
+            <img src="images/luke_skywalker_figure.jpg" alt="Luke Skywalker using the force">
+                <p>Looking around my desk, I was inspired by the figurines that give my workspace
+                    character. When I looked at the fortune cookie I noticed how beautful 
+                    reflected light can be and the magical experience it gives. When I prepared
+                    to take a picture of the fortune cookie I noticed that the reflect light 
+                    in front of Luke Skywalker makes it seem like he is using the force.
+                </p>
+        </div>
+        <div class="photo-container">
+            <img src="images/sunset_coffee.jpg" alt="sunset in a cup of coffee">
```

Comment:
Thanks for using alt tags! These go a long way in helping people who have vision impairments use our products.